mnefun/_mnefun.py

```diff
@@ -1639,9 +1640,14 @@ def gen_inverses(p, subjects, run_indices):
             for f, m, e in zip(out_flags, meg_bools, eeg_bools):
                 fwd_restricted = pick_types_forward(fwd, meg=m, eeg=e)
                 for l, s, x, d in zip(looses, tags, fixeds, depths):
+                    if l < 1:
+                        use_cps = True
+                    else:
+                        use_cps = False
```

Comment:
we should always use CPS

Reply:
So `else use_cps = None` for the fixed and free ori cases, correct?

Reply:
I would just always set `use_cps=True`

Reply:
The [docstr](https://martinos.org/mne/dev/generated/mne.minimum_norm.make_inverse_operator.html) for `make_inverse_operator` says cps is...

>Only used when converting to surface orientation (i.e., for surface source spaces and loose < 1).

Won't it balk with `loose = 1`?